app.js

```diff
@@ -4,15 +4,76 @@ const express = require('express');
 const hbs = require('hbs');
 
 // require spotify-web-api-node package here:
+const SpotifyWebApi = require("spotify-web-api-node");
+
+const spotifyApi = new SpotifyWebApi({
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+});
+
+// Retrieve an access token
+spotifyApi
+  .clientCredentialsGrant()
+  .then((data) => {
+    console.log({body: data.body})
```

Comment:
Its cool try to know if the things are working, but every time we create a Pull Request we should remove the console.logs that are not necessary, you can leave the ones that are inside catch